gengokit/generator/gen.go

```diff
@@ -76,6 +77,18 @@ func generateResponseFile(data *gengokit.Data, prevFile io.Reader, templFP strin
 		if genCode, err = applyTemplateFromPath(templFP, data); err != nil {
 			return nil, errors.Wrapf(err, "cannot render template: %s", templFP)
 		}
+	case middlewares.EndpointsPath:
```

Comment:
This must have happened in the rebase and I did not notice. Woops, thanks for the catch.